src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -50,10 +50,20 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.0.0" />
   </ItemGroup>
 
+  <!-- .NET Standard 2.0 -->
```

Comment:
SUPER NIT. I'd've expected this section before .NET Standard 1.6, as we seemed to be putting modern frameworks on the top. This is not really a big deal. Carry on!

Reply:
Sure, will do